app/controllers/admin/information_architectures/sub_categories_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Admin
+  module InformationArchitectures
+    class SubCategoriesController < BaseController
+      def create
+        category = Category.find_by(id: params[:category_id])
+        @sub_category = SubCategory.new(category_params)
+
+        if category.present? && @sub_category.save
```

Comment:
let it be so for now but 
line 8 should not be event executed if `category.present? is false ` on line 10